[2020-06-12 14:42:12,229] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: dataops.modeling 2020-06-11T00:00:00+00:00 [queued]>
[2020-06-12 14:42:12,256] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: dataops.modeling 2020-06-11T00:00:00+00:00 [queued]>
[2020-06-12 14:42:12,256] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-12 14:42:12,256] {taskinstance.py:880} INFO - Starting attempt 8 of 11
[2020-06-12 14:42:12,256] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-12 14:42:12,279] {taskinstance.py:900} INFO - Executing <Task(BashOperator): modeling> on 2020-06-11T00:00:00+00:00
[2020-06-12 14:42:12,282] {standard_task_runner.py:53} INFO - Started process 22436 to run task
[2020-06-12 14:42:12,348] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: dataops.modeling 2020-06-11T00:00:00+00:00 [running]> vinicius-Latitude-3480
[2020-06-12 14:42:12,372] {bash_operator.py:82} INFO - Tmp dir root location: 
 /tmp
[2020-06-12 14:42:12,373] {bash_operator.py:103} INFO - Temporary script location: /tmp/airflowtmpy5f71yxe/modeling1yla0qbl
[2020-06-12 14:42:12,373] {bash_operator.py:115} INFO - Running command: python /home/vinicius/Documents/Projetos/Claro/Piloto/git_Homologação/src/modeling/run.py
[2020-06-12 14:42:12,379] {bash_operator.py:122} INFO - Output:
[2020-06-12 14:43:36,849] {bash_operator.py:126} INFO - Treinando modelo KN_Test
[2020-06-12 14:43:36,849] {bash_operator.py:126} INFO - INFO: 'Test_model' does not exist. Creating a new experiment
[2020-06-12 14:43:36,850] {bash_operator.py:126} INFO - Treinando modelo KN_4
[2020-06-12 14:43:36,850] {bash_operator.py:126} INFO - Treinando modelo test_LR
[2020-06-12 14:43:36,850] {bash_operator.py:126} INFO - Tabela salva em /home/vinicius/Documents/Projetos/Claro/Piloto/git_Homologação/src/modeling/output/model_output_83816.csv
[2020-06-12 14:43:36,850] {bash_operator.py:126} INFO - dados dos modelos salvos em /home/vinicius/Documents/Projetos/Claro/Piloto/git_Homologação/src/modeling/output/model_output_83816.csv
[2020-06-12 14:43:37,151] {bash_operator.py:128} INFO - Command exited with return code 0
[2020-06-12 14:43:37,179] {taskinstance.py:1052} INFO - Marking task as SUCCESS.dag_id=dataops, task_id=modeling, execution_date=20200611T000000, start_date=20200612T174212, end_date=20200612T174337
[2020-06-12 14:43:37,756] {logging_mixin.py:112} INFO - [2020-06-12 14:43:37,755] {local_task_job.py:103} INFO - Task exited with return code 0
